### PR TITLE
Even Newer New Crystal Hollow Items

### PR DESCRIPTION
**Pull request by Strebbypatty**
Yes only one item this time, the only other thing thats not in the repo right now from this update is the 5th unknown new pet that was added with it, (has been leaked to be an ammonite maybe idk)

### CODE REVIEW

File: items/DIVAN_ALLOY.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "itemid": "minecraft:skull",
+  "displayname": "§f§f§6Divan\u0027s Alloy",
+  "nbttag": "{overrideMeta:1b,HideFlags:254,SkullOwner:{Id:\"87e89bea-b2db-3d69-8429-16f5c4b889a0\",Properties:{textures:[0:{Value:\"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOGRhNDE0ZDI5Y2M5ZWJiZmMxY2JkY2QyMTFlZWU0NzI2ZDA2NzZiZTI2MmU5Y2I4ZWVmZmFmZDFmYzM4MGIxNCJ9fX0\u003d\"}]}},display:{Lore:[0:\"§7§7§oThe legendary Divan explored\",1:\"§7§owhere no other man has set foot,\",2:\"§7§oand came back with this\",3:\"§7§oincredibly rare alloy.\",4:\"\",5:\"§6§lLEGENDARY\"],Name:\"§f§f§6Divan\u0027s Alloy\"},ExtraAttributes:{originTag:\"UNKNOWN\",id:\"DIVAN_ALLOY\",uuid:\"b46873d3-7b91-4f16-bd2c-99747fc82060\",timestamp:\"7/23/21 2:44 AM\"},AttributeModifiers:[]}",
```
Comment (jani270) [inline items/DIVAN_ALLOY.json:4]: Not removing Timestamp and uuid, bad :)
Other then cool.

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/DIVAN_ALLOY.json (hunk 1 above)
Comment (Strebbypatty) [reply items/DIVAN_ALLOY.json:4]: I am big brain

**Review by jani270 — APPROVED** [review]
epic

**Review by IRONM00N — APPROVED** [review]
if jani says its good then I say its good

**Pull request merged**